Resources/config/admin.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<container xmlns="http://symfony.com/schema/dic/services" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://symfony.com/schema/dic/services http://symfony.com/schema/dic/services/services-1.0.xsd">
+    <parameters>
+        <!-- ARTICLE -->
+        <parameter key="sonata.article.admin.article.class">Sonata\ArticleBundle\Admin\ArticleAdmin</parameter>
```

Comment:
> however, there is no particular needs this class to be configurable.

In this case just do not use neither configuration and class parameter. :wink: 

> BTW those services declarations are consistent which those in page or media bundles

Because when they was created all people did like this. The recommendation against this appeared after those bundles.